src/glsl/common.glsl

```diff
@@ -68,6 +68,8 @@ layout(std140, binding=EZQ_GL_BINDINGPOINT_FRAMECONSTANTS) uniform GlobalState {
 	float fogMinZ;
 	float fogMaxZ;
 	float padding;
+
+	vec3 camAngles;
```

Comment:
Shouldn't padding above be removed as the added vec3 will hit the next even boundary, or it's not shader struct padding, actually something else?

Reply:
yes it should i just didnt know std140 rules at the time (lol)